android/app/src/googleplay/java/org/fossasia/openevent/utils/MicrolocationClusterWrapper.java

```diff
@@ -0,0 +1,29 @@
+package org.fossasia.openevent.utils;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.google.maps.android.clustering.ClusterItem;
+
+import org.fossasia.openevent.data.Microlocation;
+
+/**
+ * Created by rohanagarwal94 on 6/7/17.
+ */
+public class MicrolocationClusterWrapper implements ClusterItem {
```

Comment:
Cant. The clustermanager needs the parameter model to implement Cluster item . [see this](https://github.com/rohanagarwal94/open-event-android/blob/4641cffc4504bc374d4de7667f3dd5e41de70aad/android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java#L56).